integration-cli/docker_cli_pull_test.go

```diff
@@ -30,11 +30,10 @@ func (s *DockerHubPullSuite) TestPullFromCentralRegistry(c *check.C) {
 
 	// We should have a single entry in images.
 	img := strings.TrimSpace(s.Cmd(c, "images"))
-	if splitImg := strings.Split(img, "\n"); len(splitImg) != 2 {
-		c.Fatalf("expected only two lines in the output of `docker images`, got %d", len(splitImg))
-	} else if re := regexp.MustCompile(`^hello-world\s+latest`); !re.Match([]byte(splitImg[1])) {
-		c.Fatal("invalid output for `docker images` (expected image and tag name")
-	}
+	splitImg := strings.Split(img, "\n")
+	c.Assert(len(splitImg), checker.Equals, 2, check.Commentf("expected only two lines in the output of `docker images`, got %d", len(splitImg)))
+	re := regexp.MustCompile(`^hello-world\s+latest`)
+	c.Assert(re.Match([]byte(splitImg[1])), checker.Equals, true, check.Commentf("invalid output for `docker images` (expected images and tag name"))
```

Comment:
this commentf can be a standard comment on the same line as there are no variables being interpolated.
```
// invalid output for `docker images` (expected images and tag name)
```
Otherwise maybe print some of the variables for debug. I wouldn't know what to print, though.